docs/getting-started/upgrading-to-v1.md

```diff
@@ -10,12 +10,56 @@ toc: true
 
 Version 1 of Bulma Clean Theme uses version 1 of Bulma. Bulma v1 has been updated to use dart sass and Jekyll was updated to use dart sass from version 4.3 and up, so this is now the minimum supported version of Jekyll for this theme. 
 
+## Updating dependencies
+
+One way of updating Jekyll and the theme is by using bundle. First update the versions in your Gemfile as follows:
+
+```ruby
+# Gemfile
+gem "jekyll", "~> 4.3"
+gem "bulma-clean-theme",  '1.0.0'
+```
+
+Then use bundle to update from your command line.
+
+```bash
+$ bundle update
+```
+
+## Using remote_theme
+
+If you are using Jekyll Remote Theme, then you can [add a version number](https://github.com/benbalter/jekyll-remote-theme?tab=readme-ov-file#declaring-your-theme) in your _config.yml to specify which version you want to use.
+
+```yaml
+# _config.yml
+remote_theme: chrisrhymes/bulma-clean-theme@v1.0.0
```

Comment:
Would you like this to be consistent with the README and installation pages? Both worked in my testing, so this isn't wrong.

```suggestion
remote_theme: chrisrhymes/bulma-clean-theme:v1.0.0
```